pmd-apex/src/main/resources/rulesets/apex/performance.xml

```diff
@@ -8,6 +8,7 @@
 The Performance ruleset contains a collection of good practices which should be followed.
   </description>
 
+    <rule ref="category/apex/performance.xml/AvoidDatabaseMethodCallsInLoops" deprecated="true" />
```

Comment:
I created entries in all xml files that contained the other "Avoid..InLoops" classes. I wasn't sure if this was correct given that they are marked `deprecated`, please let me know.

Reply:
These are old rulesets, which we don't maintain anymore. They are just here for backwards compatibility and should not be used anymore. So, just revert the change here, thanks.